package.json

```diff
@@ -28,9 +28,7 @@
     "webpack-cli": "3.3.10"
   },
   "browserslist": [
-    "> 1%",
-    "last 2 firefox versions",
-    "last 2 safari versions",
+    "defaults",
     "ie 11"
```

Comment:
who is checkinf ig IE still works :sweat_smile:

Reply:
IE 11 is already partially broken in some regards, and I don't think we got any recent bug reports about it. I've been meaning to remove its support a while ago, maybe we should do it now.

Reply:
What does it cost to keep it ?

Reply:
I've used it sometime when needing an extra session in addition to Chrome 😛.

Reply:
I guess I'll do it. Windows 7 EOL is in 2 days and Windows 10 users have Edge installed as an alternative.